README.md
# Hoffman Lab Website

This repository hosts the source for the Hoffman Lab at Harvard. The website is built in Jekyll, a static site generator.

## Setup

To get started, follow Ruby installation instructions ![here](https://jekyllrb.com/docs/installation/).

From there, clone this repo, `cd` in and run `gem install jekyll bundler`.

Then start serving the website with `bundle exec jekyll serve`. If all goes well, you should see a live-reloading copy of the site!

Google online for setting up Github pages hosting for Jekyll. 



_pages/news.md
---
title: "Hoffman Lab - News"
layout: gridlay
excerpt: "Hoffman Lab -- News"
sitemap: false
permalink: /news/
---

# News


<div class="container" style="position: relative; right: 2%">


{% for newsitem in site.data.news %}

<div class="row">
<div class="col-sm-8">
<div class="well">
{% if newsitem.image %}<img style="{{newsitem.style}}" src="{{ site.url }}{{ site.baseurl }}/images/news/{{newsitem.image}}"/>{% endif %}
<h3 style="text-align: center" data-toc-text="{{newsitem.toc}}">{{newsitem.title}}</h3>
<em>{{newsitem.description}}{% if newsitem.url %}<a class="nounder" href="{{newsitem.url}}">...read more</a>{% endif %}</em><br/>
{% if newsitem.type %}<i style="float:right;" class="material-icons">{{newsitem.type}}</i>
{% else %} <i style="float:right" class="material-icons">info</i>
{% endif %}

</div>
</div>
</div>

{% endfor %}
</div>

images/people/Postdoc.php
<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
<title>Postdoc Application</title>
<link href="/style/default.css" rel="stylesheet" type="text/css">
</head>

<body>
 <?php $date_modified="April 1, 2018"; ?>
 <?php $submenu="none"; ?>
 <?php include('../sidebar.php'); ?>

<div id="content">

<h1>Postdocs</h1>

<h3>Information Requested</h3>
<ul>
  <li>Send to Prof. Hoffman (<a href="mailto:jhoffman@physics.harvard.edu?subject=work%20in%20%your%20lab&cc=dlclarke@fas.harvard.edu">jhoffman&#64;physics.harvard.edu</a>) and cc to Dionne Clarke (<a 
href="mailto:dlclarke@fas.harvard.edu">dlclarke&#64;fas.harvard.edu</a>) with useful subject line.</li>
  <li> Cover email (include experience, interests, &amp; goals; preferred start date; etc.)</li>
  <li> CV (include hyperlink to each of your publications)</li>
  <li> Contact information for at least 2 references (full name, context &amp; duration in which they know you, email, phone number)</li>
  <li> Funding: information about any fellowship opportunities you have in mind.
  <li> If possible, please also arrange for at least one letter (from one of your above-named references) to be emailed directly to me, with your name in the subject line. This letter should come 
from somebody who knows you well in a research setting. (Note that in the US, students do not typically read their own reference letters. If a reference letter comes to me directly from a prospective employee, 
it would not be interpreted as a serious, honest letter.)
 </ul>

<h3>Funding</h3>
<ul>
 <!-- <li><a href="https://academicjobsonline.org/ajo/jobs/2773">Golub Fellowship</a></li> -->
 <li><a href="http://environment.harvard.edu/environmental-fellows-program">Harvard University Center for the Environment</a></li>
 <li><a href="http://sysbio.harvard.edu/research-fellowship.html">John Harvard Distinguished Science Fellow</a></li>
 <li><a href="http://www.socfell.fas.harvard.edu">Harvard Junior Fellow</a></li>
 <li><a href="http://us.fulbrightonline.org">Fulbright Fellowship</a></li>
 <li><a href="https://www.physics.harvard.edu/research/scholars/FAQs#fundres">list of other funding sources</a></li>
</ul>

<h3>Helpful Information</h3>
<ul>
<!-- <li> Harvard <a href="http://gsas.harvard.edu/programs-study/non-degree-programs">visiting fellows</a></li> -->
 <li> Harvard <a href="http://www.hio.harvard.edu/">international office</a></li>
 <li> Harvard <a href="https://postdoc.harvard.edu/Postdoc-benefits">benefits</a></li>
 <li> Harvard <a href="http://www.huhousing.harvard.edu/">housing</a></li>
 <li> <a href="http://boston.craigslist.org/">craigslist</a> (to find local housing off-campus)</li>
</ul>

</div>
</body>


images/people/VisitingStudent.php
<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
<title>Visiting Student Application</title>
<link href="/style/default.css" rel="stylesheet" type="text/css">
</head>

<body>
 <?php $date_modified="April 1, 2018"; ?>
 <?php $submenu="none"; ?>
 <?php include('../sidebar.php'); ?>

<div id="content">

<h1>Visiting Students</h1>

<h3>Information Requested (ideally by March 15 for summer internships)</h3>
<ul>
 <li><h5>Email with the following information</h5>
 <ul>
  <li>Send to Prof. Hoffman (<a href="mailto:jhoffman@physics.harvard.edu?subject=work%20in%20%your%20lab&cc=dlclarke@fas.harvard.edu">jhoffman&#64;physics.harvard.edu</a>) and cc to Dionne Clarke (<a 
href="mailto:dlclarke@fas.harvard.edu">dlclarke&#64;fas.harvard.edu</a>) with useful subject line.</li>
  <li> Experience, interests, &amp; goals (1-2 paragraphs)</li>
  <li> Preferred employment dates (e.g. &quot;June 15 - Aug 30&quot;. Note that a typical summer internsthip is at least 10 weeks.)</li>
  <li> Will you have your own funding? (We do not typically have funding to support external undergraduates. For reference, a typical Harvard undergraduate receives &#36;4000 for 10 weeks of summer 
employment, which gives a sense of the local cost of living.)</li>
  <li> Will you have your own health insurance? (If you're in the US, you're typically on your parents' insurance through age 26. If you're international, you may need to purchase travel health insurance, 
example links coming soon...)
  <li> Full contact information for at least 2 references (full name, context &amp; duration in which they know you, email, phone number)</li>
 </ul>
 <li><h5>Attachments</h5>
 <ul>
  <li>CV (include experience, skills, and most important coursework; see <a href="exampleCV1.pdf">example</a>)</li>
  <li>Transcript</li>
 </ul>
 <li><h5>Reference letter</h5> Please also arrange for at least one letter (from one of your above-named references) to be emailed directly to me, with your name in the subject line. Ideally this letter should come 
from somebody who knows you well in a research setting. (Note that in the US, students do not typically read their own reference letters. If a reference letter comes to me directly from a student, 
it would not be interpreted as a serious, honest letter.)
 </ul>

<h3> Some helpful information</h3>
<ul>
 <li> Harvard <a href="http://gsas.harvard.edu/programs-study/non-degree-programs">special students &amp; visiting fellows</a></li>
 <li> Harvard <a href="http://www.hio.harvard.edu/">international office</a></li>
 <li> <a href="http://boston.craigslist.org/">craigslist</a> (to find local housing)</li>
 <li> travel health insurance (coming soon...)</li>
</ul>

</div>
</body>


images/people/HarvardUndergrad.php
<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
<title>Harvard Undergraduate Application</title>
<link href="/style/default.css" rel="stylesheet" type="text/css">
</head>

<body>
 <?php $date_modified="April 1, 2018"; ?>
 <?php $submenu="none"; ?>
 <?php include('../sidebar.php'); ?>

<div id="content">

<h1>Harvard Undergraduates</h1>

<h3>Information Requested</h3>
<ul>
 <li>Send to Prof. Hoffman (<a href="mailto:jhoffman@physics.harvard.edu?subject=work%20in%20%your%20lab&cc=dlclarke@fas.harvard.edu">jhoffman&#64;physics.harvard.edu</a>) and cc to Dionne Clarke (<a 
href="mailto:dlclarke@fas.harvard.edu">dlclarke&#64;fas.harvard.edu</a>) with useful subject line.</li>
 <li>Email about your experience, interests, and goals (1-2 paragraphs. No prior research experience is necessary, but if you have some, we'd be interested to hear about it.)</li>
 <li>CV (include experience, skills, and coursework; see <a href="exampleCV1.pdf">example</a>)</li>
 <li>Unofficial transcript (not essential, if relevant coursework is already on your CV)</li>
 <li>Preferred employment dates (e.g. &quot;spring semester only&quot; or &quot;summer from June 10-Aug 20, then continuing into fall semester &amp; possibly beyond&quot;. Note that a summer internship is 
typically 10+ weeks, and a term-time commitment must be at least 10 hours/week, to maintain substantive progress.)</li>
 <li>Funding you have in hand, or plan to apply for (see below); or desired course credit (e.g. ES 91r, Physics 91r). Note that you can <em>either</em> receive pay or course credit in a 
given semester, but not both simultaneously.</li>
 </ul>

<h3>Term Time Funding (typical deadlines end of Sept &amp; Jan)</h3>
 
<ul>
<li><a 
href="http://uraf.harvard.edu/uraf-administered-programs">Harvard
College Research Program (HCRP)</a>

<li><a href="https://seo.harvard.edu/faculty-aide-program">Faculty
Aide Program</a>: Remind your professor to apply. The program offers rolling admission with no deadlines. 

<li><a href="https://uraf.harvard.edu/mmuf">Mellon
Mays</a>: research funding for sophomore minority students </li>

<li><a href="https://uraf.harvard.edu/resources">URAF</a> and <a href="https://college.harvard.edu/financial-aid/forms-resources/resources/research-and-learning">Financial Aid</a>: Review these sites for a comprehensive 
list of research funding opportunities.
</ul>

<h3>Summer Funding (typical deadlines in March)</h3>

<ul>
<li><a href="http://uraf.harvard.edu/uraf-administered-programs">Harvard University Administered Programs</a></li>

<li><a href="https://uraf.harvard.edu/prise-application-instructions">PRISE</a>: summer housing
in Leverett </li>

<li><a
href="https://uraf.harvard.edu/herchel-smith-application-instructions">Herchel
Smith</a></li>

<li><a href="https://www.seas.harvard.edu/k-12-community-programs/reu">REU</a> through
SEAS and NSEC: Email <a href="mailto:hollar@seas.harvard.edu">Kathryn Hollar</a> with additional questions</li>

<li><a href="https://uraf.harvard.edu/mmuf">Mellon
Mays</a>: research funding for sophomore minority students</li>

<li><a href="http://uraf.harvard.edu/uraf-administered-programs">Harvard
College Research Program (HCRP)</a>

<li><a href="https://college.harvard.edu/admissions/choosing-harvard/research-opportunities">Dean's
Summer Research</a>: thesis research grants for students on financial aid </li>

<li>Summer funding for physics concentrators: please email <a 
href="mailto:djmorin@fas.harvard.edu">David Morin</a>.</li>

<li><a href="http://www.environment.harvard.edu/student-resources/undergraduate-summer-research-fund"> Harvard University Center for 
the Environment (HUCE)</a></li>

<li><a href="https://seo.harvard.edu/faculty-aide-program">Faculty
Aide Program</a>: Remind your professor to apply. The program offers rolling admission with no deadlines. 

<li>Be a <a href="http://www.summer.harvard.edu/about">summer 
school proctor</a> for free housing </li> </ul>

<h3>How to ask for a letter of <a class="h3" href="/policies/recommendations.php">recommendation</a>.</h3>

</div>
</body>
